src/Core/SecretsManager/Entities/ApiKey.cs

```diff
@@ -2,22 +2,24 @@
 using Bit.Core.Entities;
 using Bit.Core.Utilities;
 
+#nullable enable
```

Comment:
⛏️ we usually put it at the top, before `using`
To be honest i don't know why 🤷 , do we want to be more consistent on this one ?